### PR TITLE
Schema error - playbook tags

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28932

## Description
Change the tags type to be mapping instead of string

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: demisto_sdk/commands/common/schemas/playbook.yml [hunk 1]
```diff
@@ -129,8 +129,11 @@ mapping:
                   simple:
                     type: text
               tags:
+                type: map
                 required: false
```
Comment (roysagi) [inline demisto_sdk/commands/common/schemas/playbook.yml:133]: you might need `allowempty` here as well - you can check that by running validate on a playbook that has this situation:
image

Comment (Shellyber) [reply demisto_sdk/commands/common/schemas/playbook.yml:133]: No need, the validation passed ok.
Ran a build using this branch
image
Failed on other stuff of course, not relevant to this,

Comment (Itay4) [reply demisto_sdk/commands/common/schemas/playbook.yml:133]: it is not needed to fix the issue, but it is needed for the case a playbook with empty tags is given
lets add it please

**Comment by Itay4** [comment]
please add changelog

**Comment by Shellyber** [comment]
@Itay4 Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed an issue where running **validate** failed on playbook with task that performs tagging.
```
Comment (Itay4) [inline CHANGELOG.md:2]:
```suggestion
* Fixed an issue where running **validate** failed on playbook with task that adds tags to the evidence data.
```

**Review by Itay4 — APPROVED** [review]
note my comment

**Pull request merged**